doc/sfp-cmis/Interface-Link-bring-up-sequence.md

```diff
@@ -184,18 +185,246 @@ if transceiver is not present:
+<br> supervisord will respawn XCVRD after termination as xcvrd is killed using SIGABRT signal
+
+```mermaid
+sequenceDiagram
+    participant OA
+    participant APPL_DB
+    participant XCVRDMT as XCVRD main thread
+    participant CmisManagerTask
+    participant DomInfoUpdateTask
+    participant SfpStateUpdateTask
+
+    activate OA
+    activate XCVRDMT
+    activate CmisManagerTask
+    activate DomInfoUpdateTask
+    activate SfpStateUpdateTask
+    OA -x OA: Crashes while handling a routine
+    deactivate OA
+    OA ->> APPL_DB : DEL PORT_TABLE
+
+    XCVRDMT -x APPL_DB : XCVRD main thread proecesses DEL event of APPL_DB PORT_TABLE
+    Note over XCVRDMT: generate_sigabrt = True
+    alt If threads > 0 are dead
+        XCVRDMT -x XCVRDMT : Kill XCVRD with SIGKILL
+    end
+    XCVRDMT -x CmisManagerTask : Stop CmisManagerTask
+    deactivate CmisManagerTask
+    XCVRDMT -x DomInfoUpdateTask : Stop DomInfoUpdateTask
+    deactivate DomInfoUpdateTask
+    XCVRDMT -x SfpStateUpdateTask : Stop SfpStateUpdateTask
+    deactivate SfpStateUpdateTask
+    Note over XCVRDMT : deinit()
+    alt self.sfp_error_event.is_set()
+        XCVRDMT -x XCVRDMT : sys.exit(SFP_SYSTEM_ERROR)
+    else if generate_sigabrt is True
+        XCVRDMT -x XCVRDMT : Kill XCVRD with SIGABRT
+
+    else
+        XCVRDMT -x XCVRDMT : Graceful exit
+    end
+    deactivate XCVRDMT
+```
+
+## Test plan and expectation
+|       Event      | APPL_DB cleared | Xcvrd restarted | Media renotify | MEDIA_SETTINGS_SYNC_STATUS value on   xcvrd boot-up for initialized transceiver | CMIS re-init triggered | Link flap |
+|:----------------:|:---------------:|:---------------:|:--------------:|:-----------------------------------------------------------------------------:|:----------------------:|:---------:|
+| Xcvrd restart    | N               | Y               | N              | MEDIA_SETTINGS_DONE                                                           | N                      | N         |
+| Pmon restart     | N               | Y               | N              | MEDIA_SETTINGS_DONE                                                           | N                      | N         |
+| Swss restart     | Y               | Y               | Y              | MEDIA_SETTINGS_DEFAULT                                                        | Y                      | Y         |
+| Syncd restart    | Y               | Y               | Y              | MEDIA_SETTINGS_DEFAULT                                                        | Y                      | Y         |
+| config   reload  | Y               | Y               | Y              | MEDIA_SETTINGS_DEFAULT                                                        | Y                      | Y         |
+| Cold reboot      | Y               | Y               | Y              | MEDIA_SETTINGS_DEFAULT                                                        | Y                      | Y         |
```

Comment:
config reload and cold reboot triggers/use-cases doesn't have link flap (until link come operationally up and then go down post bring up).
Since all SW components went down along with HW ones (NPU, PHY, optics etc.), so should mark this case  as N/A under this link flap column